src/biogeochem/CNPrecisionControlMod.F90

```diff
@@ -22,9 +22,9 @@ module CNPrecisionControlMod
 
   ! !PUBLIC DATA:
   real(r8), public :: ccrit    =  1.e-8_r8              ! critical carbon state value for truncation (gC/m2)
-  real(r8), public :: cnegcrit = -6.e+1_r8              ! critical negative carbon state value for abort (gC/m2)
+  real(r8), public :: cnegcrit = -6.e+6_r8      ! -6.e+1_r8        ! critical negative carbon state value for abort (gC/m2)
```

Comment:
cnegcrit, nnegcrit, and ncrit are all on the namelist so we might as well return these to the values on master.